bam/bam_test.go

```diff
@@ -198,23 +198,15 @@ func BenchmarkReadIndex(b *testing.B) {
 	if *findex == "" {
 		b.Skip("no index file specified")
 	}
-	b.StopTimer()
-	f, err := os.Open(*findex)
-	if err != nil {
-		b.Fatalf("Open failed: %v", err)
-	}
-
-	buf, err := ioutil.ReadAll(f)
+	buf, err := ioutil.ReadFile(*findex)
 	if err != nil {
 		b.Fatalf("Index read failed: %v", err)
 	}
-	f.Close()
+	rdr := bytes.NewReader(buf)
 
-	b.StartTimer()
+	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-
-		rdr := bytes.NewBuffer(buf)
-
+		rdr.Seek(0, os.SEEK_SET)
```

Comment:
Do we support Go versions that do not have `io.SeekStart`? It's a nicer constant.